ha-sip/src/call.py

```diff
@@ -348,6 +379,42 @@ def play_audio_file(self, audio_file: str) -> None:
         if sound_file_name:
             self.play_wav_file(sound_file_name, True)
 
+    def play_playlist(self, playlist: List[str], language: str) -> None:
+        log(self.account.config.index, 'Playing playlist: %s' % repr(playlist))
+        wav_playlist = list()
+        remove_list = list()
+        for playlist_item in playlist:
+            try:
+                item_type, content = playlist_item.split(':', 1)
+                content = content.lstrip()
+            except ValueError:
+                log(self.account.config.index, 'Playlist item has wrong format: "%s.". Must start with "message:" or "audio_file:"' % repr(playlist_item))
+                continue
+            if item_type == 'message':
+                sound_file_name, must_be_deleted = ha.create_and_get_tts(self.ha_config, content, language)
+                # log(self.account.config.index, 'playlist: adding message: %s (filename: %s)' % (repr(content), sound_file_name))
+                if must_be_deleted:
+                    remove_list.append(sound_file_name)
+            elif item_type == 'audio_file':
+                sound_file_name = audio.convert_audio_to_wav(content)
+                if not sound_file_name:
+                    log(self.account.config.index, "Sound file couldn't be converted: %s" % repr(content))
```

Comment:
Need to always add the result to the `remove_list` when the call to `convert_audio_to_wav` does not return `None`